edx_proctoring/api.py

```diff
@@ -2490,12 +2490,13 @@ def get_integration_specific_email(provider):
     return getattr(provider, 'integration_specific_email', None) or constants.DEFAULT_CONTACT_EMAIL
 
 
-def get_enrollments_for_course(course_id):
+def get_enrollments_for_course_can_take_proctored_exams(course_id, text_search=None):
```

Comment:
Why do we need the API function? Why not just use the `enrollment_service` directly in the view? I see patterns either way in edx_proctoring, so there is not standard on this, right?

Reply:
I assume it's to provide an abstraction over how data is being fetched, allowing us to keep that logic out of the view.